config/config.go

```diff
@@ -181,6 +185,16 @@ func (l *Local) validateAuthOfflineURL() error {
 	return nil
 }
 
+func (l *Local) validateAuthType() error {
+	for _, authType := range AuthTypes {
+		if l.Auth.Type == authType {
+			return nil
+		}
```

Comment:
Could you follow a similar approach to [this](https://github.com/preslavmihaylov/todocheck/pull/74/files#diff-b20fa56bdafc4db95f805c14326690bf70c3e4932928c846923fa83caa24b9beR35) for consistency's sake?

ie, having an `IsValid` function on the `AuthType`